src/index.js

```diff
@@ -74,40 +75,42 @@ export default function commonjs(options = {}) {
 				const actualId = id.slice(PROXY_PREFIX.length);
 				const name = getName(actualId);
 
-				return (extensions.indexOf(extname(id)) === -1
-					? Promise.resolve(false)
-					: getIsCjsPromise(actualId)
-				).then(isCjs => {
+				return getIsCjsPromise(actualId).then(isCjs => {
 					if (isCjs)
 						return `import { __moduleExports } from ${JSON.stringify(
 							actualId
 						)}; export default __moduleExports;`;
-					else if (esModulesWithoutDefaultExport.indexOf(actualId) !== -1)
+					else if (esModulesWithoutDefaultExport[actualId])
 						return `import * as ${name} from ${JSON.stringify(actualId)}; export default ${name};`;
+					else if (esModulesWithDefaultExport[actualId]) {
+						return `export {default} from ${JSON.stringify(actualId)};`;
+					}
 					else
```

Comment:
What's the logic for modules that neither don't have a default nor have a default?